### PR TITLE
Page next/last hot keys

### PR DESCRIPTION
**Pull request by lcflight**
-Adds a hot keys for post next and post last.
-Hot keys should not interfere with any common accessibility, operating system, or browser keyboard shortcuts.
-Post next = shift+I
-Post previous = shift+U

### CODE REVIEW

File: src/pages/page/[page].astro [hunk 1]
```diff
@@ -4,6 +4,7 @@ import getPosts from "../../lib/getPosts";
 import Posts from "../../components/Posts.astro";
 import type { Post } from "../../schemas/post";
 import Paginated from "../../layouts/Paginated.astro";
+// import hotkeys from '../../lib/hotkeys.ts';
```
Comment (narthur) [inline src/pages/page/[page].astro]: Can we delete this?

File: src/lib/__snapshots__/getPosts.spec-snapshot.ts.snap
Comment (narthur) [inline src/lib/__snapshots__/getPosts.spec-snapshot.ts.snap:1]: Why was this file deleted?

**Review by narthur — CHANGES_REQUESTED** [review]
Looking good!

File: src/pages/page/[page].astro (hunk 1 above)
Comment (lcflight) [reply src/pages/page/[page].astro]: corrected

File: src/lib/__snapshots__/getPosts.spec-snapshot.ts.snap
Comment (lcflight) [reply src/lib/__snapshots__/getPosts.spec-snapshot.ts.snap:1]: I don't understand why this is showing up as deleted. I had to update an old hash, but i didn't delete the file. my local drive has the file, and git doesn't show any new files not added.

**Review by narthur — APPROVED** [review]
lgtm!
